Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOAlertFlag.swift

```diff
@@ -0,0 +1,119 @@
+import Library
+import SwiftUI
+
+struct PPOAlertFlag: View {
+  let alert: Alert
+
+  var body: some View {
+    HStack {
+      self.image
+        .renderingMode(.template)
+        .aspectRatio(contentMode: .fit)
+        .foregroundStyle(self.foregroundColor)
+        .frame(width: Constants.imageSize, height: Constants.imageSize)
+      Spacer()
+        .frame(width: Constants.spacerWidth)
+      Text(self.alert.message)
+        .font(Font(Constants.font))
+        .foregroundStyle(self.foregroundColor)
+    }
+    .padding(Constants.padding)
+    .background(self.backgroundColor)
+    .clipShape(RoundedRectangle(cornerSize: CGSize(
+      width: Constants.cornerRadius,
+      height: Constants.cornerRadius
+    )))
+  }
+
+  var image: Image {
+    switch self.alert.type {
+    case .time:
+      Image(Constants.timeImage)
+    case .alert:
+      Image(Constants.alertImage)
+    }
+  }
+
+  var foregroundColor: Color {
+    switch self.alert.icon {
+    case .warning:
+      Color(uiColor: Constants.warningForegroundColor)
+    case .alert:
+      Color(uiColor: Constants.alertForegroundColor)
+    }
+  }
+
+  var backgroundColor: Color {
+    switch self.alert.icon {
+    case .warning:
+      Color(uiColor: Constants.warningBackgroundColor)
+    case .alert:
+      Color(uiColor: Constants.alertBackgroundColor)
+    }
+  }
+
+  struct Alert: Identifiable {
```

Comment:
We discussed this offline, but I do think this should live in a different file. I'm okay with this living here for now if it's moving once it's using real data, though!